cmake/Dependencies.cmake

```diff
@@ -1354,23 +1354,24 @@ endif()
 
 if(USE_DISTRIBUTED AND USE_TENSORPIPE)
   if(MSVC)
+    set(USE_TENSORPIPE OFF CACHE BOOL "" FORCE)
     message(WARNING "Tensorpipe cannot be used on Windows.")
   else()
     if(USE_CUDA)
       set(TP_USE_CUDA ON CACHE BOOL "" FORCE)
       set(TP_ENABLE_CUDA_IPC ON CACHE BOOL "" FORCE)
     endif()
     set(TP_BUILD_LIBUV ON CACHE BOOL "" FORCE)
-    add_compile_options(-DTORCH_USE_LIBUV)
```

Comment:
Did you remove this one because it is already in tensorpipe_uv and this was a duplicate?

Reply:
Yes